### ISSUE TITLE
Could you share the detail parameters for your best model？

### ISSUE THREAD

**Issue by JiexingQi** [shuyanzhou/docprompting#9]
Hi, @shuyanzhou ,I recently try to train from strach and the performance is always lower than you reported. Could you share the detailed parameters for your best model? Thanks a lot.

**Comment by shuyanzhou** [comment]
The hyper-parameters are listed in the launching scripts in the README. Could you share your configurations, I will try to examine them from my end.

**Comment by JiexingQi** [comment]
Thanks for your kindly reply, @shuyanzhou  my wandb project are public, you can see the parameters in this url:
https://wandb.ai/jxqi/fid?workspace=user-jxqi
And all these experments get about 33-34 BLEU in conala (CodeT5+Docprompting).

**Comment by JiexingQi** [comment]
It seems like I did not align the batch_size to you, and I will try run it on a big memory GPU again, thank you.

**Comment by JiexingQi** [comment]
@shuyanzhou  I have align all the parameters with you in this experiment :
https://wandb.ai/jxqi/fid/runs/jtaq9uzs/overview?workspace=user-jxqi
the parameters diff are as follows:
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/43906450/220504104-6c6222f8-39ab-4007-bc57-40801b1836f7.png">
The left is I used, the right is you provided,

however, the best performance is 34.5
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/43906450/220504294-c5702191-825b-4cad-a137-059fc41eb02e.png">

Looking forward to your reply. Thanks a lot.

**Comment by JiexingQi** [comment]
I try the same parameters in different machines with GPU(3090, 4090, V100, A100), And the performance seems can not reach 34.5.

<img width="753" alt="image" src="https://user-images.githubusercontent.com/43906450/220535994-b5819483-395e-4952-8c4a-84e73e6d9f9a.png">

**Comment by shuyanzhou** [comment]
Hi @JiexingQi thanks for sharing the details. These are very helpful. The training looks correct to me -- the best BLEU I got in the **development** set was 34.41, and 36.22 in the **test** set. Can you perform the evaluation on the test set to double-confirm?

**Comment by JiexingQi** [comment]
Oh, I see. I will try the performance on the test set. Thank you very much for your reply.

**Comment by JiexingQi** [comment]
I have successfully reproduce your result on test set, thanks a lot for your kindly help.
